src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/AmazonAthenaDataSourceItemFixture.cs

```diff
@@ -0,0 +1,85 @@
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class AmazonAthenaDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_WithDataSource_SetsTitleAndDataSource()
```

Comment:
Should be `Constructor_SetsTitleAndDataSource_WithDataSource` or `Constructor_SetsTitleAndDataSource_AsProvided`